### ISSUE TITLE
feat(browser): migrate unsafe antiforgery requests to htmx 4

### ISSUE THREAD

**Issue by egil** [egil/Htmxor#56]
Parent: #77

## Protected behavior

When a package-only .NET 10 Blazor static SSR application supplies htmx 4.0.0 and a component declares POST, PUT, PATCH, and DELETE handlers, Htmxor uses stock Blazor antiforgery credentials and the htmx 4 request context to validate each unsafe request before invoking the matching request-component callback and swapping its static SSR response.

## Product decision

The next Htmxor version targets htmx 4 only. It does not carry a dual-version browser adapter or retain htmx 1.x compatibility assets.

Antiforgery remains required for unsafe methods, but `HtmxHeadOutlet` must not return to serializing the removed general htmx configuration. The component supplies a stock Blazor antiforgery token through `EditForm` or `<AntiforgeryToken />`; ASP.NET Core owns its normal antiforgery cookie and validation. Htmxor's browser adapter uses htmx 4's `htmx:config:request` context to attach the request token, together with the Htmxor action identity, to unsafe requests.

Htmxor must stop issuing a separate readable request-token cookie on every response. Pages that do not render stock antiforgery UI must not lose cache eligibility merely because Htmxor is installed. A missing or invalid token fails closed at ASP.NET Core antiforgery validation before component binding, lifecycle work, or callbacks.

This resolves the original concern in this issue: the prototype's application-wide `HX-XSRF-TOKEN` response cookie is not the v1 model.

## Acceptance criteria

- Start from exact current `origin/main`; do not extend the merged issue #108 branch.
- Use a separately restored package-only `net10.0` Blazor static SSR application, real Kestrel, real Chromium, and an application-owned exact htmx 4.0.0 asset with htmx 4 defaults.
- The browser fixture owns no htmx 1.x asset, compatibility extension, or `htmx-config` antiforgery payload.
- One stock `@page` component statically declares POST, PUT, PATCH, and DELETE component callbacks and renders stock Blazor antiforgery credentials.
- A real browser interaction for each method sends the htmx 4 request, passes antiforgery validation, invokes only the matching callback on the request-created component instance, and swaps the returned static SSR HTML.
- Missing or invalid credentials are rejected before application callback effects. Preserve method and action isolation.
- A normal full-page GET remains available. Installing Htmxor does not add the prototype `HX-XSRF-TOKEN` response cookie to an unrelated GET-only page.
- Migrate the Htmxor adapter from the legacy request event/detail shape to htmx 4's standardized request context for both antiforgery and action identity.
- Remove the obsolete custom antiforgery middleware/API if the executable proof confirms it has no remaining v1 responsibility.
- Move maintained samples to their application-owned htmx 4 asset and remove the temporary legacy sample assets/configuration only after the unsafe browser proof passes.
- Preserve meaningful failing browser evidence before the production change. Run focused package/browser checks, fast and full profiles, and fresh independent Standards and Spec reviews at exact head.
- Mutation testing is optional for this proof of concept and is not a required gate.

## Out of scope

Do not add QUERY/`@onquery`, `hx-action`, `hx-method`, `hx-query`, fragment or `<hx-partial>` behavior, response-header migration, caching policy beyond removal of Htmxor's unconditional request-token cookie, TLS, package publication, or the broader framework matrix. Do not add controllers, Minimal API handlers, static endpoint callbacks, private reflection, copied renderer code, or an embedded htmx runtime.

**Comment by cristiz1992** [comment]
Looking at how Blazor handles the antiforgery token for EditForm, a possible solution would be to inject a child input to each tag that contains hx-get, hx-post, etc.
The downside would be if the tag that has the hx-post attribute does not allow child items (ex: &lt;input /&gt;).

If I look at other frameworks like [Django htmx](https://django-htmx.readthedocs.io/en/latest/tips.html), they attach to the body tag the hx-headers and all requests will send the hardcoded header with the antiforgery token.
```
<body hx-headers='{"x-csrftoken": "{{ csrf_token }}"}'>
  ...
</body>
```
The downside here is if the Data Protection key ring used to encrypt it has rotated or isn't available anymore (e.g., app redeployed without persisted keys, different machine in a farm without shared key storage, or the key's expiration window passed) the hardcoded antiforgery token is not valid anymore.

I'm thinking of a middle ground solution would be to have a meta tag with the antiforgery value (what you proposed above):
`<meta id="hxor-csrf-token" name="hxor-csrf-token" value="XXX">`
then to check the htmx antiforgery middleware if the `Set-Cookie` header was set with the new antiforgery cookie and if the request is htmx then set on response header `HXOR-CSRF-TOKEN` the new antiforgery token value and
then in htmxor.js, check the afterRequest event for that header and update the meta tag value accordingly.
```
const requestToken = document.cookie
			.split("; ")
			.find(row => row.startsWith(antiforgery.cookieName + "="))
			.split("=")[1];
```
The above will need to change to read from the meta tag instead.
